routers/dashboard.py

```
@@ -0,0 +1,1042 @@
+display_transfer_message_english = config("DISPLAY_TRANSFER_MESSAGE_ENGLISH")
+display_transfer_message_arabic = config("DISPLAY_TRANSFER_MESSAGE_ARABIC")
+
+display_bye_message_english = config("DISPLAY_HUMAN_END_MESSAGE_ENGLISH")
+display_bye_message_arabic = config("DISPLAY_HUMAN_END_MESSAGE_ARABIC")
+
+def str_to_datetime(timestamp_str):
+    return datetime.strptime(timestamp_str, "%d/%m/%Y %H:%M:%S")
+
+dashboard_router = APIRouter()
+
+def str_to_datetime(timestamp_str):
```

Ruff (0.8.2)

Redefinition of unused `str_to_datetime` from line 22

(F811)